packages/v4/src/content/design-guidelines/components/notification-drawer/notification-drawer.md

```diff
@@ -4,9 +4,7 @@ section: components
 related: ['Notification drawer', 'Notification badge', 'Drawer']
 ---
 
-The **Notification Drawer** uses the [Drawer component](/documentation/core/components/drawer) to provide a place for viewing and managing incoming notifications.
```

Comment:
Change link to be updated for new website - /components/drawer/html

Reply:
I replaced that text @mmenestr It should no longer be there.